server/repository/src/tests.rs

```diff
@@ -571,9 +570,6 @@ mod repository_test {
         // setup
         let name_repo = NameRowRepository::new(&connection);
         name_repo.insert_one(&data::name_1()).await.unwrap();
-        NameLinkRowRepository::new(&connection)
-            .upsert_one(&mock_name_link_from_name(&data::name_1()))
-            .unwrap();
```

Comment:
name_repo.insert_one already does this...

Reply:
Nice catch!